src/main/scala/org/anormcypher/Neo4jREST.scala

```diff
@@ -2,12 +2,20 @@ package org.anormcypher
 
 import play.api.http.HttpVerbs.{DELETE, POST}
 import play.api.libs.iteratee._
-import play.api.libs.json._, Json._
+import play.api.libs.json._
+import Json._
+import akka.stream.Materializer
+import akka.stream.scaladsl.Sink
+import akka.util.ByteString
+import play.api.libs.streams.Streams
 import play.api.libs.ws._
-import scala.concurrent._, duration._
+
+import scala.concurrent._
+import duration._
 
 class Neo4jREST(val wsclient: WSClient, val host: String, val port: Int,
-  val username: String, val password: String, val https: Boolean) extends Neo4jConnection {
+  val username: String, val password: String, val https: Boolean)
+  (implicit materializer: Materializer) extends Neo4jConnection {
```

Comment:
I would put the `materializer` as a val in the constructor.  In the companion's `apply` method it's okay to use the implicits since it's a factory method.  A constructor should be explicit about its dependencies so nothing gets picked up inadvertently because it happens to be in the implicit scope.